src/components/buttonPanel.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Button from './button.component'
+
+const ButtonPanel = () => {
+    const buttons = [
+        [{name: 'AC', color: ''}, {name: '+/-', color: ''}, {name: '%', color: ''}, {name: '÷'}],
```

Comment:
- [ ] Each button row should have a height of 100px

Reply:
I think you meant "each row" must be 100px height.

Reply:
Yes.